packages/ui-components/src/components/utils.ts

```diff
@@ -12,3 +12,19 @@ export const getConfig = (): UIComponentsConfig => {
 
 	return DEFAULT_CONFIG;
 };
+
+export const getCollapsedElement = (elements: NodeListOf<ChildNode>): HTMLElement => {
```

Comment:
what you think in clear a little bit this function?

my suggestion:
```
export const getCollapsedElement = (elements: NodeListOf<ChildNode>): HTMLElement | undefined => {
	if (elements.length === 0) return;

	return Array.from(elements).find(node => {
		if (getCssStyle(node as HTMLElement, 'text-overflow') === 'ellipsis') {
			return node;
		}

		return getCollapsedElement(node.childNodes);
	}) as HTMLElement;
};
```

Reply:
Won't this return the first parent of the Element found?